documentatie/use-cases/GSB-subfuncties.md

```diff
@@ -53,3 +53,20 @@ __uitbreidingen__:
 
 - De eerste stap van invoer is aangeven of er herteld is vanwege een verschil tussen aantal toegelaten kiezers en aantal uitgebrachte stemmen. Hoe verhoudt de invoer van die stap zich tot het oplossen van waarschuwingen over aantallen toegelaten kiezers en uitgebrachte stemmen?
 - Als de coördinator het PV terugstuurt in het proces, naar welk punt dan precies?
+
+
+## De coördinator bewerkt de stembureaus
+
+__niveau:__ subfunctie, vis, 🐟
+
+### Hoofdscenario en uitbreidingen
+
+__hoofdscenario__:
+
+1. De coördinator voegt stembureaus toe, past stembureaus aan en verwijdert stembureaus waar nodig.
+2. De applicatie controleert dat de bewerking aan de validaties voldoet.
+
+__uitbreidingen__:
+2a. (bij verwijderen) Er is al invoer of een resultaat voor het stembureau:
+&emsp; 2a1. De applicatie geeft aan dat het stembureau niet verwijderd kan worden.
```

Comment:
Dit zet de applicatie naar mijn gevoel nog net iets teveel in een passieve rol. Toen ik het las voelde de huidige implementatie met een alert als een perfect valide oplossing. Terwijl je eigenlijk wilt sturen op een oplossing waarbij de optie tot verwijderen er überhaupt niet is.

Voorstel:

```suggestion
&emsp; 2a1. De applicatie voorkomt dat het stembureau verwijderd kan worden.
```

Reply:
> De referentie is de lijst die voorafgaand aan de verkiezingen wordt gepubliceerd. Je wilt dat wat in Abacus staat daarmee overeen komt.

Ja eens. Maar dat moet je geregeld hebben voordat er invoer was, toch? Als er eenmaal uitslagen zijn ingevoerd voor een stembureau, moet de applicatie er wat mij betreft vanuit gaan dat er dus een papieren proces-verbaal was, en kan je het stembureau niet meer verwijderen.

Reply:
Ik heb de hele sub-functie use case verwijderd, want ofwel werd het kort, lelijk en vaag, ofwel veel te lang van wat het nu is.

Heb bij de gebruikersdoel een preconditie toegevoegd "De invoer is nog niet gestart." en ik ben nog bezig met een open punt voor diezelfde use case over hoe we dat afdwingen. Niet verwijderen van stembureaus met invoer gebeurt nu namelijk, omdat we onze database goed hebben ingericht met foreign keys. Ook checken bij aanpassen moet anders opgelost. En als we fases in Abacus gaan inbouwen, dan moet dat ook in deze validaties terechtkomen.